src/io/iter_image_recordio_2.cc

```diff
@@ -360,10 +360,10 @@ inline void ImageRecordIOParser2<DType>::ParseChunk(dmlc::InputSplit::Blob * chu
           (rand_uniform(*(prnds_[tid])) * normalize_param_.max_random_illumination * 2
           - normalize_param_.max_random_illumination) * normalize_param_.scale;
       }
+      DType RGBA[4] = {}
```

Comment:
Oops, sorry. I tested it in my machine, but updated branch through website I didn't notice.